x/trigger/simulation/genesis.go

```diff
@@ -0,0 +1,140 @@
+package simulation
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"time"
+
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	sdktx "github.com/cosmos/cosmos-sdk/types/tx"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+
+	"github.com/provenance-io/provenance/internal/pioconfig"
+	"github.com/provenance-io/provenance/x/trigger/types"
+)
+
+const (
+	TriggerID  = "trigger_id"
+	QueueStart = "queue_start"
+)
+
+// TriggerIDFn randomized MinActions
+func TriggerIDFn(r *rand.Rand) uint64 {
+	return uint64(simtypes.RandIntBetween(r, 2, 10000000000))
+}
+
+// QueueStartFn randomized MinActions
+func QueueStartFn(r *rand.Rand) uint64 {
+	return uint64(simtypes.RandIntBetween(r, 0, 10000000000))
```

Comment:
Generated Ids for objects cannot be 0 because they were following the NullObject pattern. In this module's case it was just the Triggers. The Queue was not given that restriction because the index is not an id attached to an object. This was mainly following how the Reward module did it and to keep it consistent.

Reply:
You are correct. Updated to start from 1.